src/query/query_types.rs

```diff
@@ -1,12 +1,19 @@
-#[derive(Debug)]
+#[derive(Debug, PartialEq)]
```

Comment:
We want PartialEq here for one of the tests, was passing this into a helper for testing and trying to use a match but turns out you can't match on a variable.. https://stackoverflow.com/a/72537694